### ISSUE TITLE
[Due for payment 2025-02-28] Selection gets removed after navigating to a single per diem option

### ISSUE THREAD

**Issue by mitarachim** [Expensify/App#54498]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.78-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** No
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:** N/A
**If this was caught during regression testing, add the test name, ID and link from TestRail:** Exp
**Email or phone of affected tester (no customers):** applausetester+nl895@applause.expensifail.com
**Issue reported by:** Applause Internal Team
**Device used:** Windows 11 pro/chrome
**App Component:** Workspace Settings

## Action Performed:
1. Navigate to staging.new.expensify.com
2. Create a workspace 
3. Enable per diem 
4. Import a spreadsheet for per diem 
5. Select all 
6. Click on one of the per diem rates

## Expected Result:
Selection doesn't reset. (App works consistently with the categories page)

## Actual Result:
Selection resets

## Workaround:
Unknown

## Platforms:
- [ ] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/c7fdb87e-7c0b-4f83-880b-ff46eb66b03f

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Issue Owner</summary>Current Issue Owner: @Christinadobrzyn</details>

**Comment by jasperhuangg** [comment]
@NJ-2020 Care to take on this issue as well?

**Comment by Christinadobrzyn** [comment]
Thanks @truph01 I assigned you to this issue

**Comment by grgia** [comment]
@truph01 any updates on this one?

**Comment by Christinadobrzyn** [comment]
Looks like we're still waiting on an answer to these questions https://github.com/Expensify/App/issues/49322#issuecomment-2567487741

Is that accurate @truph01?

**Comment by Christinadobrzyn** [comment]
Oh perfect thank you @truph01! It looks like @MonilBhavsar [gave](https://github.com/Expensify/App/issues/49322#issuecomment-2578483143) a  👍 to move forward with your [proposal](https://github.com/Expensify/App/issues/49322#issuecomment-2567487741). 

Sorry, can you let us know how that will affect this issue? Should we put it on hold for now?

**Comment by Christinadobrzyn** [comment]
Oh perfect thank you @truph01! It looks like @MonilBhavsar [gave](https://github.com/Expensify/App/issues/49322#issuecomment-2578483143) a  👍 to move forward with your [proposal](https://github.com/Expensify/App/issues/49322#issuecomment-2567487741). 

Sorry, can you let us know how that will affect this issue? Should we put it on hold for now?

**Comment by grgia** [comment]
@truph01 what issue is this held on?

**Comment by Christinadobrzyn** [comment]
monitoring - https://github.com/Expensify/App/issues/49322

**Comment by Christinadobrzyn** [comment]
monitoring - https://github.com/Expensify/App/issues/49322

**Comment by grgia** [comment]
linked issue still in progress

**Comment by Christinadobrzyn** [comment]
monitoring - https://github.com/Expensify/App/issues/49322

**Comment by Christinadobrzyn** [comment]
monitoring - https://github.com/Expensify/App/issues/49322

**Comment by Christinadobrzyn** [comment]
watching https://github.com/Expensify/App/pull/56667

**Comment by Christinadobrzyn** [comment]
watching https://github.com/Expensify/App/pull/56667

**Comment by Christinadobrzyn** [comment]
@truph01 do we need a regression test?

**Comment by JmillsExpensify** [comment]
$250 approved for @allroundexperts
